package.json

```diff
@@ -4,10 +4,10 @@
   "description": "A plugin for Fastify to enable management of cache control headers",
   "main": "plugin.js",
   "scripts": {
-    "test": "tap test/*.test.js",
-    "test-ci": "tap --cov test/*.test.js",
     "lint": "standard --verbose | snazzy",
-    "lint-ci": "standard"
+    "lint:ci": "standard",
+    "test": "tap test/*.test.js",
+    "test:ci": "tap --cov test/*.test.js"
```

Comment:
`test` and `test:ci` are identical, they both run the coverage.

Reply:
Glad you raised that, wanted to ask about it! Will review.

Reply:
Resolved with https://github.com/fastify/fastify-caching/pull/55/commits/d76e46ee2ed0fe87b2fe72ad1ca03a246a8b07b3